runtime/vara/src/lib.rs

```diff
@@ -1019,6 +1019,7 @@ impl pallet_gear::Config for Runtime {
     type ProgramRentEnabled = ConstBool<false>;
 
     type ProgramRentDisabledDelta = ConstU32<{ WEEKS * RENT_DISABLED_DELTA_WEEK_FACTOR }>;
+    type RentPoolId = StakingRewards;
```

Comment:
naming scares here as I said in comment for this implementation.
it looks like we use pallet id of pallet-staking-rewards for rent pool id